docker/docker-compose.dev.yml

```diff
@@ -16,6 +16,7 @@ x-server-build: &server-common
   volumes:
     - ../server:/usr/src/app
     - ${UPLOAD_LOCATION}/photos:/usr/src/app/upload
+    - ${UPLOAD_LOCATION}/photos/upload:/usr/src/app/upload/upload
     - /usr/src/app/node_modules
```

Comment:
Why are you changing this? This seems like a noop

Reply:
I'm guessing that having them as two separate mounts means they get treated as different filesystems, hence the rename operation needs to become a move, causing that to get implicitly tested during dev?

Reply:
Exactly this, I didn't actually plan to include this change, but thinking about it, it might be worth it so we're always testing the copy mechanism in dev